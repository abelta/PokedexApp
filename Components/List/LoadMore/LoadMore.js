import React from 'react';
import PropTypes from 'prop-types';
import { Text, TouchableHighlight } from 'react-native';

const LoadMore = ({ onPress }) => (
  <TouchableHighlight onPress={onPress}>
    <Text>Load more</Text>
  </TouchableHighlight>
);

LoadMore.propTypes = {
  onPress: PropTypes.func, // eslint-disable-line react/require-default-props
};

export default LoadMore;
