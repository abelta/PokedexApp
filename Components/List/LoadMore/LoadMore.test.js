import React from 'react';
import renderer from 'react-test-renderer';
import LoadMore from '.';

describe('LoadMore', () => {
  const tree = renderer.create(<LoadMore />).toJSON();
  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
