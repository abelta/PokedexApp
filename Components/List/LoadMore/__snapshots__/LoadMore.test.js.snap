// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadMore renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
      },
      undefined,
    ]
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Load more
  </Text>
</View>
`;
